modules.d/01systemd-coredump/module-setup.sh

```diff
@@ -52,6 +53,7 @@ install() {
     if [[ $hostonly ]]; then
         inst_multiple -H -o \
             "$systemdutilconfdir"/coredump.conf \
+            "$systemdutilconfdir/coredump.conf.d/*.conf" \
             "$systemdsystemconfdir/coredump.conf.d/*.conf" \
```

Comment:
L57 is clearly a typo, it should be exactly as the line you're adding.

Reply:
Right.